src/content/components/FeatureView/FeatureView.js

```diff
@@ -310,121 +426,193 @@ export class FeatureView extends React.PureComponent {
       };
     };
 
-    const predicate = () =>
-      this._isMounted && id === this.props.match.params.id;
-    await this.context.qm.runCachedQueries(
+    const predicate = () => isMounted;
+    qm.runCachedQueries(
       subMetaQuery,
```

Comment:
Curious why we need `subMetaQuery` for ResolvedView?

Reply:
The way the feature view works, both for ResolvedView and EngineeringView, is that it fetches all the blockers of the given feature bug and displays them. But some of those blockers are submetas. So it first fetches submeta blockers and uses those as the input to the next query, to get all blockers of submeta blockers of the feature bug.

EngineeringView additionally sorts the bugs by submeta, to create categories within the view. ResolvedView doesn't do any sorting by submeta, but it still needs to query the submetas since it's trying to show blockers of blockers.

I think a nicer way to do this would be some kind of recursive query, where it queries blockers of the feature meta, and if any of those blockers have the `meta` keyword, it adds them to a second query to get their blockers, and if any of _those_ blockers have the `meta` keyword, it does the same thing again; with perhaps up to 4 levels of recursion. Right now it only shows 2 levels, i.e. blockers of blockers of the feature.

But since I was just trying to restore this view ASAP so it'll be available for us, I left the query the same as it was before. Adding that kind of recursion will be more time-intensive, since I'd need to figure out visually how to represent up to 4 levels, or however many we choose to show. So probably something for a followup.